src/host/renderData.hpp

```diff
@@ -15,15 +15,12 @@ Author(s):
 #pragma once
 
 #include "../renderer/inc/IRenderData.hpp"
-#include "../types/inc/colorTable.hpp"
-#include "../types/IUiaData.h"
 
 class RenderData final :
-    public Microsoft::Console::Render::IRenderData,
-    public Microsoft::Console::Types::IUiaData
+    public Microsoft::Console::Render::IRenderData
 {
 public:
-#pragma region BaseData
+#pragma region IRenderData

```

Comment:
Is there any reason to keep this `#pragma region` around anymore? Every function in this file falls under it haha.